### PR TITLE
[10.0] Add module product_analytic_sale

### PR DESCRIPTION
**Pull request by alexis-via**
With this PR, we now have a full set : product_analytic + product_analytic_pos + product_analytic_purchase

### CODE REVIEW

File: product_analytic_sale/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+=====================
+
+This module is a glue module between the *product_analytic* and
+*sale* modules.
+
+Usage
+=====
+
+When you create an invoice from a sale order with products that have an
+**expense analytic account** (or whose internal category has an
```
Comment (feketemihai) [inline product_analytic_sale/README.rst:16]: should be income analytic account

File: product_analytic_sale/README.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+**expense analytic account**), it will be automatically set on the
```
Comment (feketemihai) [inline product_analytic_sale/README.rst:17]: the same, income analytic account

**Review by feketemihai — CHANGES_REQUESTED** [review]
@alexis-via I don't get it why do tou need this module extensions, since you only use them for invoice lines, and you already have in product_analytic https://github.com/OCA/account-analytic/blob/10.0/product_analytic/models/account_invoice.py#L32, why do you need the prepare for purchase, pos, sale since on create it's already checking if no analytic, will be added depending on the type of invoice...

**Comment by alexis-via** [comment]
@feketemihai Yes, you are probably right... this module product_analytic_sale is not needed.
The module product_analytic_pos is required because pos doesn't create invoices. The module product_analytic_purchase sets the analytic on the purchase order line also.

**Pull request closed without merging**